transcoder/message/DatacastParser.py

```diff
@@ -69,10 +74,15 @@ def process_schema(self) -> [DatacastSchema]:
         return filtered_list
 
     def _process_schema(self) -> [DatacastSchema]:
+        """Function for sub-class to implement for processing schema"""
         raise ParserFunctionNotDefinedError
 
     def process_message(self, raw_msg) -> ParsedMessage:
         """Wraps _process_message with count and inclusion behavior"""
+        if self.frame_only is True:
+            self.increment_summary_count('data')
+            return ParsedMessage.ParsedMessage('data', 'data', raw_msg, {'data': raw_msg})
```

Comment:
Let's say we wanted to take messages out of a pcap file and create a length-delimited file out of them. This would in essence strip the pcap transport semantics and IP headers and leave us with a message payload. If the unframed  bytes were sent on as the payload, we could define an output_type as length delimited that would just write sizes and bytes out pretty easily. Or a base64 encoded payload of line_delimited output types. In short, in a frame only world, my vote would be to just return `raw_msg` vs. a dictionary

Reply:
This might mean short-circuiting the entire process_message routine? or at least until it gets to the output transport

Reply:
Makes sense, though with the current output options, all output managers with exception of diag require a structured output. If we introduced a flat file output format type, we could also add conditionality for this?

Reply:
With `--frame_only`, we might consider two constraints for a first iteration.

The first being: "Pin output to STDOUT" and the second being "as a length-delimited stream".

However, from an implementation standpoint, I wonder if this means that it should have it's own OutputManager? It would almost be a hybrid of a schemaless `DiagnosticOutputManager` and a `LengthDelimitedFileMessageSink`. 

At least for a first iteration STDOUT shell redirection to a file, or piped back into another instance of `txcode` would be done. But one could imagine quickly the value of doing this with Pub/Sub as a sink. 

Maybe a longer-term thing to consider is: "do we decouple output managers and schemas?" In that case we might imagine a `LengthDelimitedOutputManager` that ignores schemas and simply emits unframed message bytes accordingly or `ByteArrayPubSubOutputManager` for the Pub/Sub sink.